acestep/audio_utils.py

```
@@ -215,7 +219,7 @@ def save_audio(
         Returns:
             Actual saved file path
         """
-        format = (format or self.default_format).lower()
+        format = "flac" 
```

Ruff (0.15.9)

[error] Variable `format` is shadowing a Python builtin

(A001)